sw/simulator/nps/nps_fdm_gazebo.cpp

```diff
@@ -83,6 +85,35 @@ static struct gazebocam_t gazebo_cams[VIDEO_THREAD_MAX_CAMERAS] =
 { { NULL, 0 } };
 #endif
 
+#ifdef NPS_SIMULATE_RANGE_SENSORS
+#include "subsystems/abi.h"
```

Comment:
this include should be inside of  extern "C" {} like on line 46. You can just add it here around the includes.